data-integration/main.bal

```diff
@@ -0,0 +1,94 @@
+import data_integration.store;
+import ballerina/http;
+import ballerina/persist;
+
+final store:Client dbClient = check new;
+
+service / on new http:Listener(9090) {
+    resource function post employees(store:EmployeeInsert employee)
+            returns http:Created|http:Conflict|http:InternalServerError {
+        int[]|persist:Error result = dbClient->/employees.post([employee]);
+        if result is persist:AlreadyExistsError {
+            return http:CONFLICT;
+        }
+        if result is persist:Error {
+            return <http:InternalServerError>{body: result.message()};
+        }
+        return http:CREATED;
+    }
+
+    resource function post task(store:TaskInsert task)
+            returns http:Created|http:Conflict|http:InternalServerError {
+        int[]|persist:Error result = dbClient->/tasks.post([task]);
+        if result is persist:AlreadyExistsError {
+            return http:CONFLICT;
+        }
+        if result is persist:Error {
+            return <http:InternalServerError>{body: result.message()};
+        }
+        return http:CREATED;
+    }
+
+    resource function get task/[int taskId]() returns store:Task|http:NotFound|http:InternalServerError {
+        store:Task|persist:Error task = dbClient->/tasks/[taskId];
+        if task is persist:NotFoundError {
+            return http:NOT_FOUND;
+        }
+        if task is persist:Error {
+            return <http:InternalServerError>{body: task.message()};
+        }
+        return task;
+    }
+
+    resource function get employeetasks/[int empId]() returns store:Task[]|http:NotFound|http:InternalServerError {
```

Comment:
I feel like the naming here isn't compatible with REST best practices. I would use `employees/[int empId]/tasks`

@MaryamZi WDYt?